src/pages/requests/RequestSummary.tsx

```diff
@@ -53,6 +64,80 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
   const [typingTimeout, setTypingTimeout] = useState<any>(null);
   const [prom, setProm] = useState<any>(Promise.resolve());
 
+  const [rowData, setRowData] = useState([
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" },
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" }
+  ]);
+
+  const [columnDefs, setColumnDefs] = useState([
+    {
+      headerName: "CMO Sample Name",
+      field: "cmoSampleName",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "CMO Patient ID",
+      field: "cmoPatientId",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Investigator Sample ID",
+      field: "investigatorSampleId",
+      sortable: true
+    },
+    {
+      headerName: "Primary ID",
+      field: "primaryId",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Preservation",
+      field: "preservation",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Tumor Or Normal",
+      field: "tumorOrNormal",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Sample Class",
+      field: "sampleClass",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Oncotree Code",
+      field: "oncotreeCode",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Collection Year",
+      field: "collectionYear",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Sample Origin",
+      field: "sampleOrigin",
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Tissue Location",
+      field: "tissueLocation",
+      sortable: true,
+      filterable: true
+    },
+    { headerName: "Sex", field: "sex", sortable: true, filterable: true }
```

Comment:
looks like these are the sample headers? Can we move these to the `helpers.tsx` or do they need to be explicitly defined here now due to the move to AgGrid?